person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(name = 'Unknown', age = nil, parent_permission: true)
+    @id = generate_id
```

Comment:
- [ ] Only `name` and `parent_permission` parameters should be optional parameters. So `age` should be required.

- [ ] You have not implemented the `generate_id` method. I cannot find this method within this class. Kindly implement it or you can use `rand(1...1000)` to generate a random number

  ```
  def initialize(age, name = 'Unknown', parent_permission: true)
    @id = rand(1..1000)
  ```